PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -529,17 +531,14 @@ extension SavedItemsListViewModel: SavedItemsControllerDelegate {
         for type: NSFetchedResultsChangeType,
         newIndexPath: IndexPath?
     ) {
-        guard .update == type else {
-            return
-        }
-
         var snapshot = buildSnapshot()
         snapshot.reloadItems([ItemsListCell<ItemIdentifier>.item(savedItem.objectID)])
         _snapshot = snapshot
     }
 
     func controllerDidChangeContent(_ controller: SavedItemsController) {
         itemsLoaded()
+        notificationCenter.post(name: .listUpdated, object: nil)
```

Comment:
just curious, why is this preferred over the other `didChange` function?

Reply:
@cyndichin `didChangeContent` is called one time once all changes (which call the other function) are performed / saved. so, instead of sending n number of notifications (for example if 10 items were added before a save), we only send 1. just a performance thing.

Reply:
makes sense, thanks for the explanation!